### DISCUSSION TITLE
Couldn't edit repo files on github.com, but worked fine to push changes from desktop: SOLVED

### DISCUSSION THREAD

**Discussion by skirsch** [community/community#62507] [Repositories]
### Select Topic Area

Product Feedback

### Body

I noticed that on my PERSONAL repositories, I could no longer edit any text using github.com web interface.

 I'd make my changes, and it would refuse to accept the commit saying:
`There was an error committing your changes: File could not be edited`

In fact, the exact message had a bizarre 0 at the end of the error message:
`There was an error committing your changes: File could not be edited0`

I went to my settings, updated my primary email (which wasn't working) to my current email address which does work, and now I can edit directly on github again in my own repo.

Hope this helps someone. Took me forever to figure this out.

`

**Comment by wxl** [comment]
So right! What's even more annoying is that some sort of *temporary* delivery issue caused GitHub to invalidate my primary email, leading to the issue.

This really needs to be fixed with a more instructive error message. That was super frustrating.

This is especially true when I'm trying to make the smallest of changes which is why I wanted to bother with the web interface at all!

**Reply by soyobat** [reply]
I’ve encountered this issue before, but only with one specific repository that I forked myself—all my other repositories can be edited normally. This is what confuses me. Since the repository has no GitHub Actions configured, I’ve already ruled out the possibility of actions locking the files. The problem still persists even now.
